profiles/sujal-goswami.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/107000806/194727763-2c517ae0-f842-4310-9f9c-c9e68f1dc433.png",
```

Comment:
I recommend more of an image like this. What do you think?

![image](https://user-images.githubusercontent.com/51391473/194772101-ae522822-b41c-4a7f-ae6c-1dd989532724.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/194772101-ae522822-b41c-4a7f-ae6c-1dd989532724.png",
```